samples/CarterSample/Features/Actors/FileForm.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Carter.OpenApi;
+
+namespace CarterSample.Features.Actors
+{
+    public class FileForm
+    {
+        public File File { get; set; }
+
+        [ApiSchemaAttributes(Format = "binary")]

```

Comment:
Does this need to be an attribute? Could this property not be deleted and then in PostFileForm it sets the Format property on RouteMetaDataRequest and then when /openapi is being called it uses the RouteMetaDataRequest.Format property to assign the Format property of the OpenApiSchema?

Reply:
The format type needs to belong to the instance of the schema, rather than the schema itself.  This implies that it is tricky to use the RouteMetaDataRequest to hold it.  In the example above, there are many instances of the type "System.String", but only one is the property of File and has binary format.